packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.0.20
```

Comment:
This should be 2.0.0 since it's a breaking change.

Reply:
done